bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreatorTest.java

```diff
@@ -77,13 +83,6 @@ private static Stream<Arguments> provideTestCasesForGetThingUIDReturnsThingUidAc
                 Arguments.of("shellyazplug-" + DEVICE_ID, SHELLYDT_PLUSPLUGSG3, "", THING_TYPE_SHELLYPLUSPLUGS), //
                 Arguments.of("shellyoutdoorsg3-" + DEVICE_ID, SHELLYDT_PLUSPLUGOUTDOORSG3, "",
                         THING_TYPE_SHELLYPLUSPLUGS), //
-                Arguments.of("shellyswitch25-" + DEVICE_ID, "", "relay", THING_TYPE_SHELLY25_RELAY), //
-                Arguments.of("shellyswitch25xx-" + DEVICE_ID, "", "relay", THING_TYPE_SHELLY25_RELAY), //
-                Arguments.of("shellyswitch25xx-" + DEVICE_ID, "", "", THING_TYPE_SHELLY25_ROLLER), //
-                Arguments.of("shellyswitch25xx-" + DEVICE_ID, "", "relax", THING_TYPE_SHELLY25_ROLLER), //
-                Arguments.of("shellyswitch26xx-" + DEVICE_ID, "", "relay", THING_TYPE_SHELLY2_RELAY), //
-                Arguments.of("shellyswitch-" + DEVICE_ID, "", "relay", THING_TYPE_SHELLY2_RELAY), //
-                Arguments.of("shellyswitch-" + DEVICE_ID, "", "", THING_TYPE_SHELLY2_ROLLER), //
```

Comment:
Were these never valid?

Reply:
tests like shellyswitch25xx- are nonsense, because they never appear
shellyswitch and shellyswitch25 do not work out, because they need the mode parameter to select the right thing type. However, we can't put shellyswitch-relay etc. in the DT/Thing Type Table, so it does no make sense to do something like shellyswitch-relay just to make the unit tests happy

Reply:
Yes, I added those testcases only to document current behavior. So with your changes, that behavior is now fixed. 👍